### PR TITLE
[bitnami/kafka]: Add affinity, tolerations, nodeSelector for kafka-exporter deployment

### PR DESCRIPTION
**Pull request by srntqn**
**Description of the change**

This PR adds ability to configure affinity, tolerations and nodeSelector for kafka-exporter.

**Benefits**

More flexible scheduling of the kafka-exporter in a cluster.

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/kafka/README.md [hunk 1]
```diff
@@ -261,8 +261,11 @@ The following tables lists the configurable parameters of the Kafka chart and th
...
-| `metrics.kafka.resources.requests`     | Kafka Exporter container resource requests                                                                                       | `{}`                                                    |
+| `metrics.kafka.resources.limits`       | Kafka Exporter container resource limits                                                                                         | `{}`
+| `metrics.kafka.resources.requests`     | Kafka Exporter container resource requests                                                                                       | `{}`                                                   |
+| `metrics.kafka.affinity`     | Affinity for Kafka Exporter pod assignment                                                                                       | `{}`
+| `metrics.kafka.nodeSelector`              | Node labels for Kafka Exporter pod assignment                                                            | `{}`   |
+| `metrics.kafka.tolerations`               | Tolerations for Kafka Exporter pod assignment                                                            | `[]`
```
Comment (rafariossaa) [inline bitnami/kafka/README.md:268]: There is a missing `|` at the end

File: bitnami/kafka/README.md [hunk 2]
```diff
@@ -261,8 +261,11 @@ The following tables lists the configurable parameters of the Kafka chart and th
...
-| `metrics.kafka.resources.limits`       | Kafka Exporter container resource limits                                                                                         | `{}`                                                    |
-| `metrics.kafka.resources.requests`     | Kafka Exporter container resource requests                                                                                       | `{}`                                                    |
+| `metrics.kafka.resources.limits`       | Kafka Exporter container resource limits                                                                                         | `{}`
+| `metrics.kafka.resources.requests`     | Kafka Exporter container resource requests                                                                                       | `{}`                                                   |
+| `metrics.kafka.affinity`     | Affinity for Kafka Exporter pod assignment                                                                                       | `{}`
```
Comment (rafariossaa) [inline bitnami/kafka/README.md:266]: There is a missing `|` at the end

File: bitnami/kafka/README.md [hunk 3]
```diff
@@ -261,8 +261,11 @@ The following tables lists the configurable parameters of the Kafka chart and th
...
 | `metrics.kafka.tlsCaCert`     | The secret key from the certificatesSecret or tlsCaSecret if 'ca-cert' key different from the default                                                   | `ca-file`                                                   |
 | `metrics.kafka.tlsCaSecret`     | Name of the existing secret containing the ca certificate file, if the ca cert can not be found in `metrics.kafka.certificatesSecret`                                        | `nil`                                                   |
-| `metrics.kafka.resources.limits`       | Kafka Exporter container resource limits                                                                                         | `{}`                                                    |
-| `metrics.kafka.resources.requests`     | Kafka Exporter container resource requests                                                                                       | `{}`                                                    |
+| `metrics.kafka.resources.limits`       | Kafka Exporter container resource limits                                                                                         | `{}`
```
Comment (rafariossaa) [inline bitnami/kafka/README.md:264]: There is a missing `|` at the end

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Hi,
  Thank you very much for this PR.
  Could you take a look to my comments ?

File: bitnami/kafka/README.md (hunk 1 above)
Comment (srntqn) [reply bitnami/kafka/README.md]: @rafariossaa fixed! Please take a look.

File: bitnami/kafka/README.md (hunk 2 above)
Comment (srntqn) [reply bitnami/kafka/README.md]: Fixed! Please take a look.

File: bitnami/kafka/README.md (hunk 3 above)
Comment (srntqn) [reply bitnami/kafka/README.md]: @rafariossaa fixed! Please take a look.

**Comment by rafariossaa** [comment]
Hi,
  Could you take a look to the README merge conflict ?

**Comment by srntqn** [comment]
@rafariossaa sorry, i didn't notice this conflict. 
I've resolved it.

**Review by rafariossaa — APPROVED** [review]
LGTM
Thanks !

**Pull request merged**